src/sfizz/CCMap.h

```diff
@@ -105,7 +107,7 @@ class CCMap {
     // typename std::vector<std::pair<int, ValueType>>::iterator begin() { return container.begin(); }
     // typename std::vector<std::pair<int, ValueType>>::iterator end() { return container.end(); }
 
-    const ValueType defaultValue;
+    ValueType defaultValue;
```

Comment:
From memory it all stems from this use case
- want to make `absl::optional` of `EGDescription` which contains `CCMap` members
- it needs `operator=`
- `operator=` is auto-deleted when the class has the copy-ctor defined
- it can't default-implement `operator=` when the member is const